README.md

```diff
@@ -25,3 +25,10 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+##Review 
+The recurrence relation I had was 3T(n/3) + n and then my time complexity is O(n log n) and I got a runtime of 4s in the succesful test, this is similar to the time complexity of merge sort. The reason for this complexity is that while the array is split into three sub-arrays instead of two, the total work done at each level of recursion is still linear ( O(n) ), and the depth of the recursion tree is logarithmic, resulting in  O(n log n)  overall. When you are asking about the work being done, it is when I am summing the 3 parts in my code, it is a constant time of O(1) since it is just the sum of 3 numbers. 
```

Comment:
If the amount of work to combine partial results is constant, where does the $+n$ in your recurrence relation come from?